apps/precursor-chemicals/views/extra-application-information.html

```diff
@@ -0,0 +1,9 @@
+{{<partials-page}}
+  {{$pageTitle}}
+    {{#t}}pages.extra-application-information.pageTitle{{/t}} &ndash; {{#t}}journey.serviceName{{/t}} &ndash; GOV.UK
```

Comment:
it worth adding in error placeholder as seen on layout template
`{{#errorlist.length}}{{#t}}errorlist.prefix{{/t}}{{/errorlist.length}} {{title}} &ndash; {{#t}}journey.serviceName{{/t}} &ndash; GOV.UK`

where `{{title}}` will be replaced with `{{#t}}pages.extra-application-information.pageTitle{{/t}}`

so that the behaviour will be the same across all the pages